IAM role permissions, addition to existing documentation (#3253)
uyuni-project/uyuni-docs: modules/specialized-guides/pages/public-cloud-guide/payg/aws/payg-aws-configure-the-image.adoc

This section covers initial preparation and configuration of the image on {aws}.

== The IAM Role

To ensure seamless operation, proper {aws} permissions for the IAM (Identity and Access Management) role are essential.
This role must be attached to the instance, otherwise disconnection errors will be thrown after one hour. 

> You can pre-create the role on your {aws} account and then select it from the existing btn:[IAM role] drop down during the instance configuration process.

Reviewer: ```suggestion
You can pre-create the role on your {aws} account and then select it from the existing btn:[IAM role] drop-down during the instance configuration process.
```

If you create the role during the creation of an instance, the role will not be attached automatically.
The role should be added post setup.

A role can be assigned to a running instance through the following steps: menu:Actions[Security > Modify IAM-role].
It needs to have the following policy attached in order to be used:

----
{
    "AttachedPolicies": [
        {
            "PolicyName": "AWSMarketplaceMeteringFullAccess",
            "PolicyArn": "arn:aws:iam::aws:policy/AWSMarketplaceMeteringFullAccess"
        }
    ]
}
----

To allow the **EC2** service to assume such role:

----
"AssumeRolePolicyDocument": {
            "Version": "2012-10-17",
            "Statement": [
                {
                    "Effect": "Allow",
                    "Principal": {
                        "Service": "ec2.amazonaws.com"
                    },
                    "Action": "sts:AssumeRole"
                }
            ]
        }
----

[NOTE]
====

> IAM Role can be attached to an **EC2** instance by means of an instance profile, which acts as a container for the Role itself.

Reviewer: ```suggestion
IAM role can be attached to an **EC2** instance by means of an instance profile, which acts as a container for the role itself.
```

.. Your organization should provide the necessary security groups and network configuration.

> .. If an IAM role has been previously created, you can attach it by selecting it from the Existing menu:IAM role[] dropdown.

Reviewer: ```suggestion
.. If an IAM role has been previously created, you can attach it by selecting it from the existing menu:IAM role[] drop-down.
```

. You will be redirected to the **EC2** Dashboard.

. Verify that the instance has successfully launched by selecting menu:Instances[] on the left sidebar.
  The {productname} {payg} instance should be running in the main content area.

[NOTE] 
.Usage and Costs
====

> Keep in mind that since this is a {payg} image, you will be billed according to your actual usage, including the number of systems you **manage** and **monitor** with this instance.

Reviewer: ```suggestion
Because this is a {payg} image, you will be billed according to your actual usage, including the number of systems you **manage** and **monitor** with this instance.
```
